File: practice/lv0/더 크게 합치기.py
# 내 풀이

def solution(a, b):
    x=str(a)+str(b)
    y=str(b)+str(a)
    if int(x)>=int(y):
        return int(x)
    else:return int(y)

# 다른 사람 풀이
def solution(a, b):
    return int(max(f"{a}{b}", f"{b}{a}"))

File: Programmer_Java/Lv0/더 크게 합치기.java
// 출처:프로그래머스
// https://school.programmers.co.kr/learn/courses/30/lessons/181939

class Solution {
    public int solution(int a, int b) {
        String c=String.valueOf(a);
        String d=String.valueOf(b);    
        if(Integer.parseInt(c+d) >Integer.parseInt(d+c)){
        return (Integer.parseInt(c+d));
        }
        
        else{
             return (Integer.parseInt(d+c));  
        }
    }
}